cmd/operator-sdk/new/cmd.go

```diff
@@ -53,48 +53,48 @@ generates a default directory layout based on the input <project-name>.
 <project-name> is the project name of the new operator. (e.g app-operator)
 `,
 		Example: `# Create a new project directory
```

Comment:
Last nit (should have mentioned this before): would be nice to have two spaces indenting each examples line:
```suggestion
		Example: `  # Create a new project directory
```
and so on.

Then the printout of examples aligns with `Usage`:
```console
$ operator-sdk new --help
The operator-sdk new command creates a new operator application and
generates a default directory layout based on the input <project-name>.

<project-name> is the project name of the new operator. (e.g app-operator)

Usage:
  operator-sdk new <project-name> [flags]

Examples:
  # Create a new project directory
  $ mkdir $HOME/projects/example.com/
  $ cd $HOME/projects/example.com/
...
```

Reply:
ok done. now I understand why it was indented before :)